src/components/shared/Reorderable.js

```diff
@@ -21,6 +21,7 @@ type Props = {|
   onChangeOrder: (number[]) => Action,
   // This forces the children to be an array of React Elements.
   // See https://flow.org/en/docs/react/children/ for more information.
+  // Be careful: children need to handle a `style` property.
```

Comment:
I'm guessing the following didn't work? There's also generics, although I'm fine not over-complicating the types here.

```
   children: React.ChildrenArray<React.Element<{ style: any }>>,
```

Reply:
I tried something similar but I never could make it work. I didn't invest more than 10 minutes though.

Element takes a type, so you need to use ElementType. I tried
```
React.ChildrenArray<React.Element<React.ElementType<{ style: Object }>>>
```
and the same with `$Supertype` but the error I got are related to the typing for `ChildrenArray` and I didn't really wanted to invest more time.